strawberry_django/fields/types.py

```diff
@@ -435,7 +435,7 @@ def resolve_model_field_type(
     elif (
         settings["GENERATE_ENUMS_FROM_CHOICES"]
         and isinstance(model_field, Field)
-        and hasattr(model_field, "choices")
+        and getattr(model_field, "choices", None)
```

Comment:
In my testing I found that `model_field.choices` was set to `None` even for `CharField`